go/integTest/shared_commands_test.go

```diff
@@ -521,3 +521,32 @@ func (suite *GlideTestSuite) TestGetDel_EmptyKey() {
 		assert.Equal(suite.T(), "key is required", err.Error())
 	})
 }
+
+func (suite *GlideTestSuite) TestDel_MultipleKeys() {
+	suite.runWithDefaultClients(func(client api.BaseClient) {
+		key1 := "testKey1"
+		key2 := "testKey2"
+		key3 := "testKey3"
```

Comment:
Consider using keys specific to tests or uuid generator for unique keys as these keys might clash with other tests and lead to unpredictable results.

Reply:
Good comment